app/tests/spec/views/settings/avatar_camera.js

```diff
@@ -197,27 +206,26 @@ function (chai, _, $, sinon, View, RouterMock, WindowMock, CanvasMock,
               view.stream.stop = function () {
                 stopped = true;
               };
-
               view.submit()
-                .then(function (result) {
+                .done(function (result) {
```

Comment:
the assertions below were not being called in my testing, refactored this a bit.